.eslintrc.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+    extends: ['airbnb', 'prettier', 'prettier/flowtype', 'prettier/react'],
```

Comment:
If we all assume this is a temporary solution: temporarily like in actively searching for a better set rules, that do not include any styling/format rules ok.

Otherwise, I oppose to keeping airbnb there.